### PR TITLE
Add Clients Service that will transform data #28

### PR DESCRIPTION
**Pull request by Oksana1788**

### CODE REVIEW

File: src/main/java/com/ipcount/core/entity/ClientEntity.java [hunk 1]
```diff
@@ -247,11 +251,11 @@ public void setDate(Date date) {
         this.date = date;
     }
 
-    public Integer getnDog() {
+    public Integer getNDog() {
         return nDog;
     }
 
-    public void setnDog(Integer nDog) {
+    public void setNDog(Integer nDog) {
```
Comment (yurii-polishchuk) [inline src/main/java/com/ipcount/core/entity/ClientEntity.java]: The previous version was good, please check out the Java naming convention and this post:
https://dertompson.com/2013/04/29/java-bean-getterssetters/
```
public String getiMessageId() {
  return iMessageId;
}

public void setiMessageId(String iMessageId) {
  this.iMessageId = iMessageId;
}
```

File: src/main/java/com/ipcount/core/service/ClientService.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import com.ipcount.core.repository.ClientRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class ClientService {
+    @Autowired
```
Comment (yurii-polishchuk) [inline src/main/java/com/ipcount/core/service/ClientService.java]: Please do not use `field injection`. Feel free to search on field/setter/constructor injection in Spring. For example: http://stackoverflow.com/questions/39890849/what-exactly-is-field-injection-and-how-to-avoid-it

Comment (yurii-polishchuk) [inline src/main/java/com/ipcount/core/service/ClientService.java]: http://vojtechruzicka.com/field-dependency-injection-considered-harmful/

File: src/main/java/com/ipcount/core/service/ClientService.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    private ClientRepository clientRepository;
+
+    public List<ClientDTO> findAllClients() {
+        Iterable<ClientEntity> clients = clientRepository.findAll();
+        List<ClientDTO> list = new ArrayList<>();
+        clients.forEach(client -> {
+            ClientDTO clientDTO = new ClientDTO();
```
Comment (anMelnyk) [inline src/main/java/com/ipcount/core/service/ClientService.java]: @Oksana1788 please, extract ClientDTO creation into builder class to make service more readable.

Comment (Oksana1788) [inline src/main/java/com/ipcount/core/service/ClientService.java]: @anMelnyk Sorry  but i didn't see your comment. I will fix it.

Comment (anMelnyk) [inline src/main/java/com/ipcount/core/service/ClientService.java]: @Oksana1788 no problem:) here is a link that might help U http://codereview.stackexchange.com/questions/9619/parsing-entity-classes-to-dto . U can implement it as util or mapper/builder- your choice.

**Comment by Oksana1788** [comment]
@yurii-polishchuk and @anMelnyk  I added maven-plugin. Is it mistake or not?

**Pull request merged**